### PR TITLE
hardhead ert

### PR DESCRIPTION
**Pull request by John-Dang**
its been 2000 years but i finally posted this while suffering gallstones, new black damage ERT for the green ordeals and any other breaches that need BLACK DAMAGE to handle them. meet the hardhead boys.

## Changelog
:cl:
add: Added new things
del: Removed old things
tweak: tweaked a few things
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

File: code/game/objects/items/ego_weapons/special.dm [hunk 1]
```diff
@@ -68,3 +70,339 @@
...
+	icon_state = "fireaxe0"
+	lefthand_file = 'icons/mob/inhands/weapons/axes_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/axes_righthand.dmi'
+	name = "drumhead axe"
+	desc = "When the company was demolished, a tribunal was held. Few were spared, many axes dulled."
+	force = 5
+	damtype = BLACK_DAMAGE
+	armortype = BLACK_DAMAGE
+	var/faction_bonus_force = 15
+	var/nemesis_factions = list("green_ordeal") // Deal more damage to Green Ordeal, wrecking crew style!
```
Comment (EgorDinamit) [inline code/game/objects/items/ego_weapons/special.dm:86]: Added variables should be at the bottom.

File: code/game/objects/items/ego_weapons/special.dm [hunk 2]
```diff
@@ -68,3 +70,339 @@
...
+	worn_icon_state = "wrench"
+	lefthand_file = 'icons/mob/inhands/equipment/tools_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
+	flags_1 = CONDUCT_1
+	slot_flags = ITEM_SLOT_BELT
+	force = 10
+	damtype = BLACK_DAMAGE
+	armortype = BLACK_DAMAGE
+	var/faction_bonus_force = 15
+	var/nemesis_factions = list("green_ordeal") // UNLOCK YOUR COVER AND STATE LAWS!
```
Comment (EgorDinamit) [inline code/game/objects/items/ego_weapons/special.dm:168]: Same here.

File: code/game/objects/items/ego_weapons/special.dm [hunk 3]
```diff
@@ -68,3 +70,339 @@
...
+	lefthand_file = 'icons/mob/inhands/equipment/tools_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
+	usesound = 'sound/items/crowbar.ogg'
+	flags_1 = CONDUCT_1
+	slot_flags = ITEM_SLOT_BELT
+	force = 10
+	damtype = BLACK_DAMAGE
+	armortype = BLACK_DAMAGE
+	var/faction_bonus_force = 15
+	var/nemesis_factions = list("green_ordeal") // UNLOCK YOUR COVER AND STATE LAWS!
```
Comment (EgorDinamit) [inline code/game/objects/items/ego_weapons/special.dm:216]: And again.

File: code/game/objects/items/ego_weapons/special.dm [hunk 4]
```diff
@@ -68,3 +70,339 @@
...
+	slot_flags = ITEM_SLOT_BACK
+
+	attribute_requirements = list(
+							FORTITUDE_ATTRIBUTE = 30,
+							JUSTICE_ATTRIBUTE = 40
+							)
+
+//////////////////////////////Hardhead Clipboard//////////////////////////////
+
+/obj/item/ego_weapon/clipboard
```
Comment (EgorDinamit) [inline code/game/objects/items/ego_weapons/special.dm:279]: You don't have to copy the entire goddamn clipboard code to this. Just keep it a weapon, remove all the other stuff.

**Review by EgorDinamit — COMMENTED** [review]
You also accidentally edited changelogs. Revert it.

**Comment by dsmith328** [comment]
I know you're fucked up but I do hope for your healthy return man. Get well soon.

**Comment by EgorDinamit** [comment]
When you have time to finish it, ping me on Discord to re-open PR.

**Pull request closed without merging**